src/index.js

```diff
@@ -20,7 +20,7 @@ class UglifyJsPlugin {
 
     const {
       uglifyOptions = {},
-      test = /\.js$/i,
+      test = /.((js)?)(\?[a-z0-9]+)?$/i,
```

Comment:
The `.` needs to be escaped (?)

```diff
- /.js
+ /\.js...
```

Why `((js)?)` ('nested' capturing groups) ?